### PR TITLE
Supporting LargeListArray in Fletcher

### PR DESCRIPTION
**Pull request by marc9595**
`FletcherDType.is_list` returns `True` for ListArrays and LargeListArrays. The convention is only when using parrow Arrays that we should distinguish between ListArrays and LargeListArrays.

### CODE REVIEW

File: fletcher/base.py [hunk 1]
```diff
@@ -198,7 +199,9 @@ def _is_numeric(self):
     @property
     def is_list(self):
         """Check if the array is a ListArray."""
-        return pa.types.is_list(self.arrow_dtype)
+        return pa.types.is_list(self.arrow_dtype) or pa.types.is_large_list(
```
Comment (artemru) [inline fletcher/base.py]: why don't put it on FletcherDtype class ?

File: fletcher/base.py [hunk 2]
```diff
@@ -893,7 +896,7 @@ def pandas_from_arrow(
         data = OrderedDict()
         for name, col in zip(arrow_object.column_names, arrow_object.itercolumns()):
             if pa.types.is_dictionary(col.type) or (
-                col.null_count == 0 and not pa.types.is_list(col.type)
+                col.null_count == 0 and not FletcherDtype(col.type).is_list
```
Comment (artemru) [inline fletcher/base.py]:
```suggestion
                col.null_count == 0 and not col.dtype.is_list
```

Comment (marc9595) [reply fletcher/base.py]: Here `col` is a `pa.ChunkedArray`, it has no `dtype`attribute

**Comment by artemru** [comment]
you can rebase it, I also add settings with no deadline.

**Comment by marc9595** [comment]
> you can rebase it, I also add settings with no deadline.

Done and the tests work. Merging.

**Pull request merged**